aten/src/ATen/native/cuda/SummaryOps.cu

```diff
@@ -289,5 +316,16 @@ Tensor _bincount_cuda(
   });
 }
 
+Tensor _hist_cuda(
```

Comment:
A good way to test this would to be to hook it up to the histc in [native_functions.yaml](https://github.com/pytorch/pytorch/blob/1f871f126fc59d3ea8c435868eed741170e959b3/aten/src/ATen/native/native_functions.yaml#L2879-L2882). After you've done this, you can call torch.histc on a cuda tensor in python, something like:
```
import torch
x = torch.tensor([0, 0.5, 1, 1.5], device='cuda')
print(torch.histc(x, ...))
```